lib/views/after_auth_screens/profile/profile_page.dart

```diff
@@ -370,69 +372,70 @@ class ProfilePage extends StatelessWidget {
                           height: SizeConfig.screenWidth! * 0.05,
                         ),
                         ElevatedButton(
+                          key: const Key('DONATE'),
                           onPressed: () async {
                             ///required fields for donation transaction
```

Comment:
commented the below code because we need the actual API key to test BraintreeDropInRequest. Currently we have mentioned `<YOUR_BRAINTREE_SANDBOX_API_KEY>` in the application as a placeholder which holds no value. Moreover the UI provided by Braintree is also not working properly in the application due to this placeholder.